CHANGES.rst

```diff
@@ -194,6 +194,9 @@ resample
 
 - Propgagate error and variance arrays in ``ResampleStep`` for imaging data. [#6036]
 
+- Record ``pixel_scale_ratio`` from ``ResampleStep`` in header keyword PXSCLRT
+  or ``meta.photometry.pixel_scale_ratio``, for later use in aperture photometry. [#6044]
```

Comment:
Need to update this now to `meta.resample.pixel_scale_ratio`

Reply:
Done.